seed/challenges/intermediate-bonfires.json

```diff
@@ -113,6 +113,7 @@
         "convert(36);"
       ],
       "MDNlinks": [
+	"Roman Numerals",
```

Comment:
The indention seems to be incorrect here. Give it a `git commit --amend` and I'll give it a `git merge`. You might have to read into tabs vs. spaces and make sure your editor uses the right one (the right one being spaces afaik).